packages/smst-to-text/lib/toText.ts

```diff
@@ -381,22 +381,11 @@ const renderTestLineOrCondition = (
 	const status = node.status ? renderTextNode(renderStatus(node.status)) : '';
 	const title = node.title.map(renderTextNode).join('');
 	const body = node.children.map(child => renderNode(child, renderTextNode, {prefix: prefix + tab, verbosity})).join('');
-	const docs = verbosity === 'verbose' && (node as TestLineNode).docs
-		? ' '.repeat(status.length) + 'docs: ' + renderNode((node as TestLineNode).docs!, renderTextNode, {verbosity})
-		: '';
-	const excludedMessage = node.status === 'excluded'
```

Comment:
@AntonSBilan I'm not sure why excluded was handled differently from other status messages. Do you remember the reason for it? I moved it to the same place where the rest of messages are processed.

Reply:
Unfortunately I don't

Reply:
So does it mean there will be no docs link for test lines without result?, asking because from your comment in ticket
` Doc links should be visible for non-error lines as well. Including the line definitions only, without the result. I.e. If translation has "verbose" mode - always display the link.` and we translate lines in js api before sending request

Reply:
Yes. I tested the new JS API version with different use cases and also prepared a few updates there. Will do a PR there as well after translation module is updated.